app/manifest/v2/chrome.json

```diff
@@ -4,5 +4,5 @@
     "matches": ["https://metamask.io/*"],
     "ids": ["*"]
   },
-  "minimum_chrome_version": "66"
+  "minimum_chrome_version": "78"
```

Comment:
Should probably update this to at least 80, so that we can use the optional chaining operator.